docs/release_notes/v1.0.0-rc.1.md

```diff
@@ -0,0 +1,397 @@
+
+- **UPDATED** opentelemetry library to 0.7.1. **CHANGED** repo ID of spring repo to spring [382](https://github.com/dapr/java-sdk/issues/382)
+
+### Python SDK
+
+- **ADDED** W3C trace headers to Dapr API calls [97](https://github.com/dapr/python-sdk/issues/97)
+
+- **Updated** dapr python sdk pkg documents [125](https://github.com/dapr/python-sdk/issues/125)
+
+- **ADDED** missing etag to StateResponse [128](https://github.com/dapr/python-sdk/issues/128)
+
+- **ADDED** Added codecov reports to python-sdk PRs [133](https://github.com/dapr/python-sdk/issues/133)
+
+### Documentation
+
+- **UPDATED** environment-setup.md [841](https://github.com/dapr/docs/pull/841)
+
+- **ADDED** Performance section in docs [863](https://github.com/dapr/docs/issues/863)
+
+- **FIXED** documentation link to "Sending Messages to Output Bindings" [867](https://github.com/dapr/docs/pull/867)
+
+- **ADDED** Documentation for enabling Dapr to talk to an app via an SSL connection [868](https://github.com/dapr/docs/issues/868)
+
+- **FIXED** dead link to Ignite video session [878](https://github.com/dapr/docs/pull/878)
+
+- **FIXED** links in page developing-applications/ides/vscode-remotecontainers/ [887](https://github.com/dapr/docs/issues/887)
+
+## Upgrading to Dapr 1.0.0-rc.1
+
+To upgrade to this release of Dapr, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have installed. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if docker command needs sudo:
+
+```bash
+dapr uninstall --all
+```
+
+For RC releases like this, download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have installed the CLI, run:
+
+```bash
+dapr init --runtime-version=1.0.0-rc.1
+```
+
+Wait for the update to finish,  ensure you are using the latest version of Dapr(1.0.0-rc.1) with:
+
+```bash
+$ dapr --version
+
+CLI version: 1.0.0-rc.1
+Runtime version: 1.0.0-rc.1
+```
+
+### Kubernetes
+
+#### Upgrading from previous version
+
+If you previously installed Dapr using Helm, you can upgrade Dapr to a new version.
+If you installed Dapr using the CLI, go [here](#starting-fresh-install-on-a-cluster).
+
+##### 1. Get the latest CLI
+
+Download the latest RC release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+##### 2. Upgrade existing cluster
+
+Upgrading from 0.11.x to 1.0.0-rc.1 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0-rc.1 placement service. These two commands disable `helm upgrade` to uninstall 0.11.x placement service.
```

Comment:
`These two commands disable `helm upgrade` to uninstall 0.11.x placement service.`

I don't understand what this sentence means, can we rephrase?

Reply:
> Upgrading from 0.11.x to 1.0.0-rc.1 safely requires additional step to keep 0.11.x placement service to migrate from 0.11.x placement service to 1.0.0-rc.1 placement service. These two commands disable `helm upgrade` to uninstall 0.11.x placement service.

changed to 

> To upgrade from 0.11.x to 1.0.0, run the below commands to migrate from 0.11.x placement service to 1.0.0 placement service: